tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
 	"compilerOptions": {
-		"target": "ES2021",
-		"module": "commonjs",
-		"lib": ["ES2021", "DOM"],
+		"module": "Node16",
+		"moduleResolution": "Node16",
+		"target": "ES2022",
+		"lib": ["ES2022"],
```

Comment:
According to https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping#node-16 it should be

```
		"target": "ES2021",
		"lib": ["ES2021"],
```